api/test/run_test.sh

```diff
@@ -18,8 +18,7 @@ succeeded=$?
 if [[ $succeeded -eq 0 ]]; then
 	printf "${green}:-) All tests passed${endcol}\n"
 else
-    printf "Full system logs:\n"
-    docker-compose logs
```

Comment:
So apparently this has never worked for `just api-test` because `docker-compose` isn't available within the container :upside_down_face:

Reply:
Ahhh, I use `api-testlocal` because it's easier to interface with `pdb` than having to `docker attach openverse-web-1` or whatever.